signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/OrganizerSignupMBean.java

```diff
@@ -1458,4 +1459,41 @@ public String getCurrentUserEmailAddress() {
 	public String getDisplayTimeFromInstant(Instant instant) {
 		return signupMeetingService.getUsersLocalDateTimeString(instant);
 	}
+	
+	public int getNumSigned(List<TimeslotWrapper> lista, int partOrWait) {
+		int retorno = 0;
```

Comment:
Same as before, change by numSigned or something like that

Reply:
Done